matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/verification/DefaultVerificationService.kt

```diff
@@ -927,6 +927,24 @@ internal class DefaultVerificationService @Inject constructor(
         updatePendingRequest(existingRequest.copy(
                 readyInfo = readyReq
         ))
+
+        notifyOthersOfAcceptance(readyReq.transactionId, senderId, readyReq.fromDevice)
+    }
+
+    /**
+     * Gets a list of device ids excluding the current one.
+     */
+    private fun getOtherDeviceIds(): List<String> = cryptoStore.getUserDevices(userId)?.keys?.filter { it != deviceId }.orEmpty()
+
+    /**
+     * Notifies other devices that the current verification transaction is being handled by [fromDeviceId].
+     */
+    private fun notifyOthersOfAcceptance(transactionId: String, otherUserId: String, fromDeviceId: String) {
+        val deviceIds = getOtherDeviceIds().filter { it != fromDeviceId }
+        val transport = verificationTransportToDeviceFactory.createTransport(null)
+        deviceIds.forEach { deviceId ->
+            transport.cancelTransaction(transactionId, otherUserId, deviceId, CancelCode.Accepted)
```

Comment:
I think this line will send multiple to-device requests(1 per device). On iOS we were able to add all the cancellations to the same contentMap and send a single request. Web does the same. I think it should be possible to change [cancelRequest](https://github.com/vector-im/element-android/blob/2761b35b6f95979be685fb270c10ed71213e33aa/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/verification/VerificationTransportToDevice.kt#L173) signature to allow you pass multiple deviceId targets.